sale_warehouse_calendar/models/sale_order_line.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from datetime import timedelta
+
+from odoo import models
+
+
+class SaleOrderLine(models.Model):
+    _inherit = "sale.order.line"
+
+    def _get_workload(self, customer_lead):
+        return 1 + customer_lead - self.company_id.security_lead
```

Comment:
any comment regarding adding `1`?

Reply:
This is to convert a delay into numbers of days.
https://en.wikipedia.org/wiki/Off-by-one_error#Fencepost_error

Reply:
So a comment is welcome

Reply:
It has been done already…